### ISSUE TITLE
Register the extension on pecl

### ISSUE THREAD

**Issue by stof** [BitOne/php-meminfo#22]
Making the extension available on pecl would make installation even easier, as it would be a oneliner

**Comment by BitOne** [comment]
Yes, you're right @stof. Thanks for the tip, will do.

I need to check how to distribute the analyzers with Pecl, and if that would even make sense.

**Comment by WyriHaximus** [comment]
This would be great. Also it would be great if the analyzer would be submitted to packagist so we can get updates through that.

**Comment by futureweb** [comment]
Is there any progress here? Request is over 3 years old now :-(

**Comment by BitOne** [comment]
Yes, I know :( I've sent the requested items and got a positive answer. But I still don't have a PECL account. I asked several times already in the last 24 months, but no answer from the PECL team.

If anybody knows someone there that could help us, that would be nice...

**Comment by WyriHaximus** [comment]
Maybe @remicollet can help out

**Comment by remicollet** [comment]
See https://pecl.php.net/account-request.php

1st step is a small intro on the ML about you and your extension
(and don't see anything in my mail archive)

**Comment by BitOne** [comment]
Thanks a lot @remicollet for jumping in! As you can see from Pecl mail archive https://www.mail-archive.com/pecl-dev@lists.php.net/msg05104.html I already ask for an account in November 2017.
I got an email in December 2017 telling me that my account has been opened, and I can log on the Pecl website, but I didn't get any username or password.

I requested these information several times without getting any answer. If you're able to give me some more information on how to connect to this account, that would be wonderful!!

**Comment by enumag** [comment]
@BitOne Seeing the account request page @remicollet linked above it seems you have chosen both the username and the password when submitting the form - there are fields for both.

**Comment by enumag** [comment]
Also for the analyzer, I see a composer.json file but in the repo but the package is not available on Packagist. Could you make that happen? I would like to install the analyzer using `composer global require bitone/php-meminfo-analyzer` to have it available globally.
